README.md

```diff
@@ -474,6 +475,40 @@ const Glimmer: AcceptsAFooString = class extends GlimmerComponent<{ Args: { foo:
 const Bound: AcceptsAFooString = /* the result of `{{component "ember-component"}}` */
 ```
 
+### With JS rather than TS
+
+If you have not yet migrated your project to TypeScript or you are in the process of migrating your project, the Glint CLI and language server will still function properly. All the Glint setup steps are identical, except that to receive equivalent rich editor support for component JS files in your project, you will need to document your components with valid JSDoc. For example:
+
+```js
+// SimpleComponent.js
+
+import Component from '@glint/environment-glimmerx/component';
+import { hbs } from '@glimmerx/component';
+import { helper } from '@glint/environment-glimmerx/helper';
+
+const or = helper(
+  /** @param {[a: *, b: *]} param */
+  ([a, b]) => a || b
```

Comment:
Not sure if it's worth getting this exact in an example like this, but writing out the type parameters for this helpler would allow the language server to understand that `{{or}}`'s return value will be the type of one of its inputs (inferring `A | B` as the return type):

```js
const or = helper(
  /**
   * @template A
   * @template B
   * @param {[a: A, b: B]} positional
   */
  ([a, b]) => a || b
);
```